cmd/internal/cli/singularity.go

```diff
@@ -196,7 +196,7 @@ func createConfDir(d string) {
 		if os.IsExist(err) {
 			sylog.Debugf("%s already exists. Not creating.", d)
 		} else {
-			sylog.Fatalf("Error attempting to create %s: %s", d, err)
+			sylog.Warningf("Could not create %s: %s", d, err)
```

Comment:
I don't see why this is even a warning, I think it should just be a debug statement.  It can be perfectly normal behavior to have a non-writable home directory so I don't think there's anything to warn about.